src/pages/surveys/outro.page.tsx

```diff
@@ -0,0 +1,59 @@
+import { Player } from '@lottiefiles/react-lottie-player';
+import type { NextPage } from 'next';
+import { useRouter } from 'next/router';
+import { useContext, useEffect } from 'react';
+
+import PageLoader from 'components/PageLoader';
+import useSession from 'hooks/useSession';
+import { ACTIONS, StoreContext } from 'store';
+
+const REDIRECT_AFTER_MS = 3000;
+
+const SurveyOutro: NextPage = () => {
+  useSession();
+  const router = useRouter();
+
+  const {
+    store: { currentSurvey },
+    dispatchAction,
+  } = useContext(StoreContext);
+
+  useEffect(() => {
+    if (currentSurvey === null) {
+      router.push('/');
+
+      return;
+    }
+
+    setTimeout(() => {
+      dispatchAction({
+        type: ACTIONS.SET_CURRENT_SURVEY,
+        value: null,
+      });
+
+      router.push('/');
+    }, REDIRECT_AFTER_MS);
+  }, []);
```

Comment:
```suggestion
  }, [currentSurvey, dispatchAction]);
```
Should we add these 2 dependencies to the `useEffect` hook? 🤔  Or are these dependencies stateless? (I don't remember 100% for `useContext` values :P )

Reply:
Updated in [51c4370](https://github.com/hanam1ni/react-surveyor/pull/94/commits/51c4370952ca8c214d23d0eed113ee984e23c7fe)